pytracking/tracker/dimp/dimp.py

```diff
@@ -116,20 +114,20 @@ def track(self, image, info: dict = None) -> dict:
         scores_raw = self.classify_target(test_x)
 
         # Localize the target
-        translation_vec, scale_ind, s, flag = self.localize_target(scores_raw, sample_pos, sample_scales)
-        new_pos = sample_pos[scale_ind,:] + translation_vec
+        translation_vec, scale_ind, s, flag, score = self.localize_target(scores_raw, sample_pos, sample_scales)
```

Comment:
As far as I understand your didn't change the localize_target return values, so it will throw an exception.
Am I wrong? do you use this function [track()] ?

Reply:
OK I noticed you changed localize_advanced. 
I don't like the idea that in case of different dimp params (advanced_localization=False) we will crash.

Reply:
good point, i need to think how to fix it

Reply:
i added  a None return when advanced_localization=False